src/VisualStudio/Setup.Next/AssemblyRedirects.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.Shell;
+using Roslyn.VisualStudio.Setup;
+
+[assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.EditorFeatures.Next.dll")]
+[assembly: ProvideBindingRedirection(
```

Comment:
Yes.  I will make note of it in the bug I file to get the extensiblity team to make a real package for it.